planning/behavior_velocity_planner/src/scene_module/run_out/dynamic_obstacle.cpp

```diff
@@ -70,6 +70,37 @@ pcl::PointCloud<pcl::PointXYZ> applyVoxelGridFilter(
 
   return output_points;
 }
+
+pcl::PointCloud<pcl::PointXYZ> extractObstaclePointsWithinPolygon(
+  const pcl::PointCloud<pcl::PointXYZ> & input_points, const Polygons2d & polys)
+{
+  namespace bg = boost::geometry;
+
+  if (polys.empty()) {
+    return input_points;
```

Comment:
If the polygon is empty for some reason, is the calculation time increased by having all input points in the calculation?
What about returning empty output_points?

Reply:
yes, calculation time would increase significantly.
I modified to return the empty points, and added the warning because the empty polygon is not expected behavior.
https://github.com/autowarefoundation/autoware.universe/pull/1073/commits/ced90d0e29a6436e0f2f10b39d35a29f9bc8a897